benchmarks/templates/benchmarks/tutorials/tutorial_deepdive_3.html

```diff
@@ -0,0 +1,42 @@
+{% extends "benchmarks/components/app-view.html" %}
+{% load static %}
+
+{% block banner %}
+<h1>Deep Dive 3</h1>
+<p>Submitting a Plugin via Github Pull Request </p>
+{% endblock %}
+
+{% block info_section %}
+  {% include "benchmarks/tutorials/tutorial-info-section.html" %}
+{% endblock %}
+
+
+{% block content %}
+<div class="box leaderboard-table-component">
+    <div class="columns is-tablet is-variable is-1-tablet">
+        <div class="column is-one-half">
+            <img class="tutorial_dd3" src="{% static "/benchmarks/img/tutorials/deep_dive_3_1.png" %}" />
+        </div>
+        <div class="column is-one-half tutorial_text">
+            <h3 class="benefits_heading is-size-3-mobile">Custom Models</h3>
+            <p class="benefits_info is-size-5-mobile">
+                Brain-Score also supports the submission of plugins via a Github Pull Request (PR).
+                For a complete description of what the required files are, as well as what to include in
+                each, please refer to <a href="http{% if request.is_secure %}s{% endif %}://{{ request.get_host }}/tutorial/deepdive_3">Deep Dive 1</a>.
+            </p>
+            <p class="benefits_info is-size-5-mobile">
+                To submit via PR, simply open a Pull Request with your submission on the appropriate domain's Github
+                repository. We hope to support automatic PR merging soon. However,
```

Comment:
>We hope to support automatic PR merging soon.

This is probably not going to happen (to handle automatic merging from forks, we'd need to turn on a setting that significantly reduces security, so not a good trade-off). Maybe remove?